app/src/main/java/org/simple/clinic/patient/PatientRepository.kt

```diff
@@ -508,4 +508,17 @@ class PatientRepository @Inject constructor(
           }
         }
   }
+
+  fun isPatientDefaulter(patientUuid: UUID): Observable<Boolean> {
+    return configProvider
+        .map { it.periodSinceLastVisitToDefaultPatient }
+        .switchMap {
+          database
+              .patientDao()
+              .isPatientDefaulter(
+                  patientUuid = patientUuid,
+                  lastRecordedBpThreshold = LocalDate.now(utcClock).minusDays(it.days.toLong())
```

Comment:
FYI: The `java.time` classes let you work directly with `Duration` and `Period`. You can do

```kotlin
LocalDate.now(utcClock).minus(periodSinceLastVisitToDefaultPatient)
```

instead of converting it to days and subtracting it as a primitive.

Reply:
Feel free to do this in the PR that completes this feature.